_includes/live-demos/drive-plugin-browse-controls/index.html

```diff
@@ -0,0 +1,3 @@
+<textarea id="drive-browse-example">
+  This demo shows you how to call tinydrive from within the tinymce api.
```

Comment:
Casing for tinydrive and tinymce? I don't know what the standards are for drive, but the editor should be referred to as `TinyMCE` in all docs and other communications I believe. Probably should also be `API` instead of `api` but I'm not sure if we have any special casing rules there beyond general "it's an acronym" things

Reply:
This is the casing we're using in the existing live demos, so I'm not entirely sure?

Reply:
These should definitely be using the correct casing. Maybe fix them in the new ones and log an issue for the existing ones?